src/lib/bootloader/grub2base.rb

```diff
@@ -227,6 +216,20 @@ def serial_console?
 
   private
 
+    def propose_grub_default
```

Comment:
I would move the code closer to other propose private methods at https://github.com/yast/yast-bootloader/pull/672/files#diff-5797ed1429ad469f007496587dfc38e58cb54f82bc44c3bf99d81b9bef6b58beR338